<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EasyOne</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="8">
     <widget class="QTableView" name="tableView">
      <property name="editTriggers">
       <set>QAbstractItemView::AllEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="filter"/>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="year"/>
    </item>
    <item row="1" column="6">
     <widget class="QPushButton" name="del_button">
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="7">
     <widget class="QPushButton" name="new_button">
      <property name="text">
       <string>New</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="Date">
      <item>
       <property name="text">
        <string>Today</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>January</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>February</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>March</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>April</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>May</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>June</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>July</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>August</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>September</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>October</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>November</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>December</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QPushButton" name="refresh_button">
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
